client-spark/common/src/main/java/org/apache/spark/shuffle/RssShuffleHandle.java

```diff
@@ -28,6 +28,7 @@
 import org.apache.uniffle.common.ShuffleServerInfo;
 
 public class RssShuffleHandle<K, V, C> extends ShuffleHandle {
+  private static final long serialVersionUID = -5519528926859572710L;
```

Comment:
I think we need to define this to `0L` like what Spark does currently. 

> Zero means "compute the version id at runtime by applying the standard algorithm" based on the actual class that you are serializing / deserializing. That means that whenever your code's effective serialization signature changes, the serialization / deserialization code will use a different version id

Refer to https://stackoverflow.com/a/21851309/9217742.